.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,111 @@
+name: Bug Report
+description: Create a bug report to help us improve MonoGame.
+body:
+- type: markdown
+  attributes:
+    value: |
+      If this is a general question about MonoGame, please ask on the [community site](https://monogame.net/community) or [join the discord](https://discord.gg/monogame).
+- type: checkboxes
+  id: prerequisites
+  attributes:
+    label: Prerequisites
+    options:
+      - label: I have verified this issue is not present in the `develop` branch
+        required: true
+      - label: I have searched [open](https://github.com/MonoGame/MonoGame/issues) and [closed](https://github.com/MonoGame/MonoGame/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported.
+        required: true
+- type: input
+  id: monogame-version
+  validations:
+    required: true
+  attributes:
+    label: MonoGame Version
+    placeholder: MonoGame 3.8.1.303
+    description: Which version of MonoGame does this bug occur in.
+- type: dropdown
+  id: monogame-platform
+  validations:
+    required: true
+  attributes:
+    label: Which MonoGame platform are you using?
+    options:
+      - MonoGame Android Application (mgandroid)
+      - MonoGame Cross-Platform Desktop Application (mgdesktopgl)
+      - MonoGame Windows Universal CoreApp Application (mguwpcore)
+      - MonoGame Windows Universal XAML Application (mguwpxaml)
+      - MonoGame Windows Desktop Application (mgwindowsdx)
+      - MonoGame iOS Application (mgios)
+      - MonoGame Game Library (mgnetstandard)
+      - MonoGame Content Pipeline Extension (mgpipeline)
+      - MonoGame Shared Library Project (mgshared)
```

Comment:
Add N/A option, sometimes its a tooling issue.

Also I don't think we need library templates in here, they aren't really platforms and could be counted under N/A.

Reply:
Changes made